rdrf/rdrf/services/io/defs/importer.py

```diff
@@ -463,7 +466,17 @@ def _create_registry_objects(self):
         if "metadata_json" in self.data:
             metadata_json = self.data["metadata_json"]
             if self._check_metadata_json(metadata_json):
-                r.metadata_json = self.data["metadata_json"]
+                as_json = json.loads(metadata_json)
+                if RegistryFeatures.PATIENT_FORM_DOCTORS in as_json:
+                    del as_json[RegistryFeatures.PATIENT_FORM_DOCTORS]
+                    if "features" in as_json:
```

Comment:
- `metadata.patient_form_doctors` used to be a number (0 is off), so you have to check that as well
- you can use `setdefault` to simplify the code a bit


```
if as_json(RegistryFeatures.PATIENT_FORM_DOCTORS, 0):
   features = as_json.setdefault("features", [])
   features.append(RegistryFeatures.PATIENT_FORM_DOCTORS)
```

Reply:
Updated.